package.json

```diff
@@ -10,11 +10,14 @@
 	},
 	"main": "init.js",
 	"scripts": {
-		"api-dev": "NODE_PATH=./client:./ node init.js --watch ./server --watch ./utils --watch ./client --watch ./node_modules/@pubpub",
-		"api-prod": "NODE_PATH=./client:./ node init.js",
+		"api-dev": "npm run watch-server & NODE_PATH=./dist/server/client:./dist/server:./dist node init.js --watch ./dist/server --watch ./node_modules/@pubpub",
```

Comment:
**Important change** — `NODE_PATH`s were updated to point to the new dist directories. Automatic reloading of the server still works as well.